.env.example

```
@@ -1,6 +1,9 @@
 # Put your secret key here, make sure to not leak this key.
 SECRET_KEY=your_secret_key
 
+# Put your API Key here
+API_KEY=your_api_key
```

API_KEY is too generic and we might be using other APIs in later time. Change this to something else for example,

```suggestion
TMDB_API_KEY=your_api_key
```